### PR TITLE
Add more examples to podlove demo page

### PR DESCRIPTION
**Pull request by alexander-heimbuch**
Includes more fixtures for demos

### CODE REVIEW

File: src/components/info/Info.vue [hunk 1]
```diff
@@ -90,21 +92,21 @@
...
-      font-size: 1.2rem;
+      font-size: 20px;
       font-weight: 300;
     }
 
     .podlove-info--subtitle {
       max-height: 3.6rem;
-      font-size: 0.9rem;
-      line-height: 1.2rem;
+      font-size: 14px;
```
Comment (plutonik-a) [inline src/components/info/Info.vue]: @alexander-heimbuch Why did you change the rem units to pixels?

Comment (alexander-heimbuch) [inline src/components/info/Info.vue]: I want to be as close as possible to @markusstuhr sketch template.

**Comment by alexander-heimbuch** [comment]
@plutonik-a ok will split up content commits next time 👍
